src/cryptenroll/cryptenroll.c

```diff
@@ -636,7 +676,15 @@ static int prepare_luks(
         assert(ret_cd);
         assert(!ret_volume_key == !ret_volume_key_size);
 
-        r = crypt_init(&cd, arg_node);
+        if (!arg_node) {
+                r = resolve_auto_node(&auto_node);
+                if (r == -EMEDIUMTYPE)
+                        return log_error_errno(r, "Block device backing /var isn't a LUKS2 device. Please specify a device node manually");
```

Comment:
we have this rule that we suffix all paths with / that refer to a directory. in docs and in log messages. hence /var → /var/ please.

also, please end the 2nd sentence in a full stop